src/main/java/redis/clients/jedis/SslVerifyMode.java

```diff
@@ -0,0 +1,22 @@
+package redis.clients.jedis;
+
+/**
+ * Enumeration of SSL/TLS hostname verification modes.
+ */
+public enum SslVerifyMode {
+
+    /**
+     * No verification at all.
```

Comment:
I would love if we put a clear indications that this should never be used in production.
I know it seems obvious, but I want to make sure it is written down.

Eg.

```suggestion
     * No verification at all. Avoid using this in production environments, poses serious security risks.
```